m2k/python/precision_adc_tutorial/trace_ad7124_filter_with_m2k.py

```diff
@@ -166,7 +167,7 @@ def capture_data(ad7124):  # Let's move the plotting out of this routine
 
 # Set a default ip address if none given as a command line argument
 # hardcoded_ip = "ip:192.168.0.235" # Example if you want to hardcode a different address
-hardcoded_ip = "ip:localhost" # Default to localhost if no argument given
+hardcoded_ip = "ip:169.254.174.92" #"ip:analog.local" # Default to localhost if no argument given
```

Comment:
not sure if we should specify custom ip address in a generic script example. I'd stick to `ip:analog.local`

Reply:
Wow, thanks @amiclaus . Fixed.